packages/zoe/src/contracts/multipoolAutoswap/pool.js

```diff
@@ -94,23 +94,25 @@ export const makeAddPool = (zcf, isSecondary, initPool, centralBrand) => {
         );
         return pool.getCentralAmountMath().make(result);
       },
+      // price in central tokens required to gain outputValue secondary units
       getCentralToSecondaryOutputPrice: outputValue => {
```

Comment:
Oof, this is still confusing to me, and I think I wrote this originally. What if we merge these four methods back into just two (`getInputPrice` and `getOutputPrice`) and instead of passing in a `value`, pass in a full amount and choose the pathway based on the brand. I think that would be more robust. 

Either way, I think we should not pass in just a value - we should pass in a full amount and check the brand.

Reply:
switching to amounts is a really good idea. As we discussed, I'm going to change the functions to return a pair of amounts. {inputAmount, outputAmount}

In the getOutputPrice case this makes sense because the required input might be worth more than the minimum wanted.  

In the getInputPrice case, the caller will sometimes intend to liquidate everything and so won't care, but other times if they hear that selling 20 simoleans will get you 6 moola, but you get the same amount if you provide 19, that improvement would be helpful.